filter_lists/regional.json

```diff
@@ -572,7 +572,7 @@
     {
         "uuid": "DABC6490-70E5-46DD-8BE2-358FB9A37C85",
         "url": "https://easylist-downloads.adblockplus.org/bitblock.txt",
-        "title": "BitBlock List (Дополнительная подписка фильтров)",
+        "title": "BitBlock",
```

Comment:
would be good to call this one out as a Russian list too, i.e. `BitBlock Russian`